src/app/api/add-pet/route.ts

```diff
@@ -0,0 +1,18 @@
+import { sql } from "@vercel/postgres"
+import { NextResponse } from "next/server"
+
+export async function GET(request: Request) {
+  const { searchParams } = new URL(request.url)
+  const petName = searchParams.get("petName")
+  const ownerName = searchParams.get("ownerName")
+
+  try {
+    if (!petName || !ownerName) throw new Error("Pet and owner names required")
+    await sql`INSERT INTO Pets (Name, Owner) VALUES (${petName}, ${ownerName});`
+  } catch (error) {
+    return NextResponse.json({ error }, { status: 500 })
+  }
+
+  const pets = await sql`SELECT * FROM Pets;`
```

Comment:
We use the `sql` template literal to construct queries directly. This isn't unlike what we'd do in traditional react endpoints.

As a reminder, template literals in javascript are items wrapped in back ticks. ` `